src/pages/storage/forms/DuplicateVolumeForm.tsx

```diff
@@ -40,10 +42,15 @@ const DuplicateVolumeForm: FC<Props> = ({ volume, close }) => {
   const eventQueue = useEventQueue();
 
   const { data: projects = [], isLoading: projectsLoading } = useProjects();
-
+  const { data: pool } = useStoragePool(volume.pool);
   const { data: volumes = [], isLoading: volumesLoading } =
     useLoadCustomVolumes(volume.project);
 
+  const isRemoteStorage = ["ceph", "cephfs", "powerflex"].includes(
```

Comment:
This should reuse constants from `src/util/storageOptions.tsx`. Also might be good to have this array and check in the util file instead of keeping it local in this component.